src/main/java/de/tum/in/www1/artemis/web/rest/TextExerciseResource.java

```diff
@@ -169,7 +186,9 @@ public TextExerciseResource(TextExerciseRepository textExerciseRepository, TextE
             result.getExampleSubmissions().forEach(exampleSubmission -> exampleSubmission.setTutorParticipations(null));
         }
 
-        if (notificationText != null) {
+        // TODO: Should we leave this activated in case the exercise is changed during the exam?
```

Comment:
for now I would suggest that changing an exercise during the exam is not possible

Reply:
I will remove the TODO